server/routes/createDummyUserRoutes.js

```diff
@@ -0,0 +1,20 @@
+let db = require('../models');
```

Comment:
Let's discuss this one before we commit?  Granted, I think we already planned to discuss this stuff.  

I like the rest of the PR though :)